scripts/nitro-testnode.sh

```diff
@@ -17,15 +17,15 @@ do
       HAS_DETACH=true
       shift
       ;;
-    -down|--shutdown)
-      docker container stop "$(docker container ls -q --filter name=nitro-testnode)"
+    -q|--quit)
```

Comment:
I am not a fan of `-q/--quit`; IMHO `-down` is better.
Even in `docker compose` you have `down` instead of `quit`.

Reply:
in docker compose `down` is not a flag. It is a subcommand and looks like this: `docker compose up` and `docker compose down`.
We should pass to this script short and long flags. But `-down` flag is already quite long:)

Reply:
Ok, can agree on this.